effectly/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,19 @@
+import { HeaderWrapper, Logo, LogOutBtn } from "./Header.styles";
+import { auth } from "../../core/firebase/firebase";
+
+interface HeaderProps {
+  className: string;
+}
+
+const Header = ({ className }: HeaderProps) => {
+  const signOut = () => auth.signOut();
+
+  return (
+    <HeaderWrapper className={className}>
+      <Logo>effectly</Logo>
+      <LogOutBtn onClick={signOut}>Log out</LogOutBtn>
```

Comment:
Maybe better would be to move that const to file with strings.

Reply:
Done with [c4327f7](https://github.com/wooczak/effectly/pull/8/commits/c4327f7a9b13ae67004aceb8e349558aee8cef2e)